content/docs/getting-started/installing.md

```diff
@@ -12,57 +12,80 @@ Scratch Addons is available in both Chrome Web Store and Add-ons for Firefox.
 - Add-ons for Firefox (for Mozilla Firefox)  
   https://addons.mozilla.org/firefox/addon/scratch-messaging-extension/  
 
-## From GitHub
-
-[The releases tab](https://github.com/ScratchAddons/ScratchAddons/releases) contains all builds of Scratch Addons, as well as the mirror of the store builds hosted on both servers.
 
 ## From source
 
+### About GitHub releases
+
+[The releases page](https://github.com/ScratchAddons/ScratchAddons/releases) contains the code and installation files for all development builds of Scratch Addons, as well as the mirror of the store builds.
+
 ### Cloning the repository
 
-This is the recommended way to install for development purposes. This assumes you have Git installed.
+This is the recommended way to install Scratch Addons for development purposes. This assumes you have Git installed.
 
-To download the repository, simply clone `https://github.com/ScratchAddons/ScratchAddons.git`. Here's an example.
+To download the repository, simply clone `https://github.com/ScratchAddons/ScratchAddons.git`.
 
 ```sh
 $ git clone https://github.com/ScratchAddons/ScratchAddons.git
 ```
+To update Scratch Addons, first `cd` into its folder, and then run the following commands.
+
+```sh
+$ git fetch
+$ git pull
+```
+
+This will update Scratch Addons and get it ready for code editing. Note that you will need to see the finish updating section [here](https://scratchaddons.com/docs/getting-started/installing/#install-on-google-chrome) if you are using Google Chrome.
```

Comment:
Turn this into /getting-started/... Without the scratchaddons part. If that works...

Reply:
> 
> 
> Turn this into /getting-started/... Without the scratchaddons part. If that works...

@TheColaber OK.